### PR TITLE
Added exercise 1

### PR DESCRIPTION
**Pull request by breabanmihaiE4**

### CODE REVIEW

File: Exercise1.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<?php
+// http://localhost/FII-Practic-PHP/Exercise1.php?a=-8&b=6&c=2
+ 
+$urlPage = "http://$_SERVER[HTTP_HOST]$_SERVER[REQUEST_URI]";
+echo "URL-ul paginii este: " . $urlPage . "<br>";
+echo "<br>";
+ 
+// Exercitiu 1
```
Comment (SemenciucCosmin) [inline Exercise1.php:8]: Remove redundant comment

File: Exercise1.php [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+print_r("Exercitiul 1: <br>");
+ 
+$numberOfGET = count($_GET);
```
Comment (SemenciucCosmin) [inline Exercise1.php:11]: Rename this. Maybe "argumentCount" or something to truly reflect its usage

File: Exercise1.php [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+if ($numberOfGET < 3) {
```
Comment (SemenciucCosmin) [inline Exercise1.php:12]: What is 3? Extract in a constant with a proper name

File: Exercise1.php [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    print_r("Nu exista suficiente variabile globale GET");
```
Comment (SemenciucCosmin) [inline Exercise1.php:13]: Change this message

File: Exercise1.php [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+} else {
+    $values = [];
```
Comment (SemenciucCosmin) [inline Exercise1.php:15]: What is values?  Rename

File: Exercise1.php [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+    foreach ($_GET as $key => $value) {
+        $values[$key] = $value;
+    }
+ 
+    $numericPositions = array_values($values);
```
Comment (SemenciucCosmin) [inline Exercise1.php:20]: Rename

File: Exercise1.php [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+ 
+    print_r($numericPositions[0] . "<br>");
```
Comment (SemenciucCosmin) [inline Exercise1.php:22]: Instead of 0, 1, 2 use constants like "FIRST_ARGUMENT"

File: Exercise1.php [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+    print_r($numericPositions[1] . "<br>");
+    print_r($numericPositions[2] . "<br>");
+ 
+    $sumOfFirst3Numbers = $numericPositions[0] + $numericPositions[1] + $numericPositions[2];
```
Comment (SemenciucCosmin) [inline Exercise1.php:26]: If there is only on sum computed rename it just as "sum". Apply this rule to the other computed values

File: Exercise1.php [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+    $multiplatOfFirst3Numbers = $numericPositions[0] * $numericPositions[1] * $numericPositions[2];
+    $divOfFirst3Numbers = $numericPositions[0] / $numericPositions[1] / $numericPositions[2];
+    $ridicareLaPutere = $numericPositions[0] ** $numericPositions[1] ** $numericPositions[2];
+ 
+    print_r("Suma primelor 3 elemente este = " . $sumOfFirst3Numbers . "<br>");
+    print_r("Diferenta primelor 3 elemente este = " . $difOfFirst3Numbers . "<br>");
+    print_r("Produsul primelor 3 elemente este = " . $multiplatOfFirst3Numbers . "<br>");
+    print_r("Impartirea primelor 3 elemente este = " . $divOfFirst3Numbers . "<br>");
+    print_r("Ridicarea la putere a primelor 3 elemente este = " . $ridicareLaPutere . "<br>");
+    print_r(8 / 6 / 5 . "<br>");
```
Comment (SemenciucCosmin) [inline Exercise1.php:37]: Do you need this to be printed? What does it mean?

File: Exercise1.php [hunk 10]
```diff
@@ -0,0 +1,39 @@
...
+    print_r($numericPositions[1] . "<br>");
+    print_r($numericPositions[2] . "<br>");
+ 
+    $sumOfFirst3Numbers = $numericPositions[0] + $numericPositions[1] + $numericPositions[2];
+    $difOfFirst3Numbers = $numericPositions[0] - $numericPositions[1] - $numericPositions[2];
+    $multiplatOfFirst3Numbers = $numericPositions[0] * $numericPositions[1] * $numericPositions[2];
+    $divOfFirst3Numbers = $numericPositions[0] / $numericPositions[1] / $numericPositions[2];
+    $ridicareLaPutere = $numericPositions[0] ** $numericPositions[1] ** $numericPositions[2];
+ 
+    print_r("Suma primelor 3 elemente este = " . $sumOfFirst3Numbers . "<br>");
```
Comment (SemenciucCosmin) [inline Exercise1.php:32]: Change message according to the above rule. You shouldn't specify the number of arguments it it is fixed

**Review by SemenciucCosmin — CHANGES_REQUESTED** [review]

File: Exercise1.php (hunk 4 above)
Comment (breabanmihaiE4) [reply Exercise1.php]: Done!

File: Exercise1.php (hunk 5 above)
Comment (breabanmihaiE4) [reply Exercise1.php]: Done!

File: Exercise1.php (hunk 6 above)
Comment (breabanmihaiE4) [reply Exercise1.php]: Done!

File: Exercise1.php (hunk 7 above)
Comment (breabanmihaiE4) [reply Exercise1.php]: Done!

File: Exercise1.php (hunk 8 above)
Comment (breabanmihaiE4) [reply Exercise1.php]: Done!

File: Exercise1.php (hunk 9 above)
Comment (breabanmihaiE4) [reply Exercise1.php]: Done!

File: Exercise1.php (hunk 10 above)
Comment (breabanmihaiE4) [reply Exercise1.php]: Done!

File: Exercise1.php (hunk 3 above)
Comment (breabanmihaiE4) [reply Exercise1.php]: Done!

File: Exercise1.php (hunk 2 above)
Comment (breabanmihaiE4) [reply Exercise1.php]: Done!

File: Exercise1.php (hunk 1 above)
Comment (breabanmihaiE4) [reply Exercise1.php]: Done!

**Review by SemenciucCosmin — APPROVED** [review]

**Pull request merged**